app/src/main/java/com/google/android/samples/socialite/ui/camera/CameraViewModel.kt

```diff
@@ -72,6 +72,8 @@ class CameraViewModel @Inject constructor(
 
     val chatId: Long? = savedStateHandle.get("chatId")
     var viewFinderState = MutableStateFlow(ViewFinderState())
+    private var _imageCaptureState = MutableStateFlow(ImageCaptureState.PENDING)
+    val imageCaptureState: SharedFlow<ImageCaptureState> = _imageCaptureState
```

Comment:
```suggestion
    val imageCaptureState: StateFlow<ImageCaptureState> = _imageCaptureState.asStateFlow()
```

Reply:
Done (minus the call to `asStateFlow()` to match the rest of the code)

Reply:
I'll address a follow-up in #44.